### PR TITLE
[WIP] Changes that require ocrd >= 2.0.0a1

### PR DESCRIPTION
**Pull request by kba**
Do not merge until 2.0.0 is stable.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,17 +9,16 @@ WORKDIR /build-ocrd
...
 RUN apt-get update && \
     apt-get -y install --no-install-recommends \
-    ca-certificates \
-    make \
-    git
-COPY Makefile .
-RUN make deps-ubuntu
-COPY ocrd_tesserocr ./ocrd_tesserocr
+    libtesseract-dev \
+    libleptonica-dev \
```
Comment (stweil) [inline Dockerfile]: `libleptonica-dev` is redundant (it is already a dependency of `libtesseract-dev`).

File: Dockerfile [hunk 2]
```diff
@@ -9,17 +9,16 @@ WORKDIR /build-ocrd
...
+    tesseract-ocr-eng \
```
Comment (stweil) [inline Dockerfile]: `tesseract-ocr-eng` is redundant, too.

File: Dockerfile [hunk 3]
```diff
@@ -9,17 +9,16 @@ WORKDIR /build-ocrd
...
+COPY Makefile .
 RUN apt-get update && \
     apt-get -y install --no-install-recommends \
-    ca-certificates \
-    make \
-    git
-COPY Makefile .
-RUN make deps-ubuntu
-COPY ocrd_tesserocr ./ocrd_tesserocr
+    libtesseract-dev \
```
Comment (stweil) [inline Dockerfile]: Does this install 4.1 (current stable release)? If it still installs 4.0 (no longer supported, known bugs), that should be changed.

The `ENV` statements in lines 5 and 6 were needed for Tesseract 4.0. Tesseract 4.1 does not require them.

File: README.md [hunk 4]
```diff
@@ -8,59 +7,83 @@ ocrd\_tesserocr
...
------
+```sh
+docker pull ocrd/tesserocr
+```
+
+### From git 
+
+This is the best option if you want to change the source code or install the latest, unpublished changes.
+
+We strongly recommend to use a [virtualenv](https://virtualenv.pypa.io/en/stable/userguide/).
```
[the remark this answers is not in the document]
Comment (kba) [reply README.md]: Good point, fixed.

File: Dockerfile (hunk 1 above)
Comment (kba) [reply Dockerfile]: Thanks, will remove.

File: Dockerfile (hunk 2 above)
Comment (kba) [reply Dockerfile]: True, thanks.

File: Dockerfile (hunk 3 above)
Comment (kba) [reply Dockerfile]: Yes, this will install [4.1.0-1](https://www.ubuntuupdates.org/package/core/eoan/universe/base/tesseract)

**Pull request merged**